benchmark/index.mjs

```diff
@@ -0,0 +1,107 @@
+import { readFile, writeFile } from 'fs/promises'
+import colorthief from 'colorthief'
```

Comment:
I think you're missing some of those dependencies (`colorthief`, `tinycolor2`, `lodash-es`) from your `devDependencies`.